_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -38,9 +38,9 @@ pipeline:
           region: us-east-1
           sts_role_arn: arn:aws:iam::123456789012:role/Data-Prepper
         max_retries: 5
-        bucket: mys3bucket
+        bucket: bucket_name
         object_key:
-          path_prefix: my-logs/%{yyyy}/%{MM}/%{dd}/
+          path_prefix: my-elb/%{yyyy}/%{MM}/%{dd}/
```

Comment:
I think we should stick with `my-logs` here. This example is not specifically for ELB logs.